provider/common/disk.go

```diff
@@ -13,7 +13,7 @@ import (
 // data.
 func MinRootDiskSizeGiB(os jujuos.OSType) uint64 {
 	switch os {
-	case jujuos.Ubuntu, jujuos.CentOS, jujuos.OpenSUSE:
+	case jujuos.Ubuntu, jujuos.CentOS:
```

Comment:
Seems like there is a quick win here to get rid of all this code and just `return 8` and also `_ jujuos.OSType`